Sources/SubprocessMocks/MockProcess.swift

```diff
@@ -95,7 +95,7 @@ open class MockProcessReference: Process {
         var standardInput: Any?
         var standardOutput: Any?
         var standardError: Any?
-        var terminationHandler: ((Process) -> Void)?
+        var terminationHandler: (@Sendable (Process) -> Void)?
```

Comment:
To support building against earlier versions of Foundation, I believe adopting the following will help.

```
#if compiler(>=5.7)
        var terminationHandler: (@Sendable (Process) -> Void)?
#else
        var terminationHandler: ((Process) -> Void)?
#endif
```

It is not a pretty way to do things as we really need to know the version of Foundation, but my research indicates this is the closest way to do as of earlier this year.